src/interface/src/app/plan/scenario-details/outcome/outcome.component.ts

```diff
@@ -59,7 +59,7 @@ export class OutcomeComponent implements OnInit, OnChanges {
     if (this.plan) {
       this.totalPlanningAreaAcres = calculateAcres(this.plan.planningArea!);
     }
-    this.authService.loggedInUser$.pipe(take(1)).subscribe(this.currentUser$);
+    //this.authService.loggedInUser$.pipe(take(1)).subscribe(this.currentUser$);
```

Comment:
This component is getting deleted on another PR so we don't really care about this change